src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ToGerritRunListener.java

```diff
@@ -428,15 +429,19 @@ private String obtainFailureMessage(@Nullable GerritTriggeredEvent event,
                 // The filename may contain environment variables
                 filepath = envVars.expand(filepath);
 
-                // Check for ANT-style file path
-                FilePath[] matches = this.getMatchingWorkspaceFiles(build.getWorkspace(), filepath);
-                logger.debug("Found matching workspace files: {}", matches);
+                if (build instanceof AbstractBuild) {
```

Comment:
So now I have a little bit better an idea what you are talking about :smile:. Next question: where specifically is this reflected in the UI i.e. in which Jelly files and on which config entries? Maybe `buildFailureMessage` in `src/main/resources/com/.../hudsontrigger/GerritTrigger/config.jelly?`